avaframe/com1DFA/com1DFA.py

```diff
@@ -1093,6 +1085,7 @@ def initializeFields(cfg, dem, particles):
     fields['FV'] = PFV
     fields['P'] = PP
     fields['FD'] = FD
+    fields['FM'] = FD
```

Comment:
general comment: do we need to initialize PFV, PP, FD, TA as 2D array of 0s they are all the same here with same dimension, so isn't one enough?